contexts/useYearn.tsx

```diff
@@ -50,11 +55,14 @@ export const YearnContextApp = ({children}: {children: ReactElement}): ReactElem
 	** anomalies.
 	**********************************************************************/
 	const getYearnDataSync = React.useCallback(async (_chainID: number): Promise<void> => {
-		const	[fromAPI, _ledgerSupport, _riskFramework] = await Promise.all([
+		const	[fromAPI, _ledgerSupport, _riskFramework, _metaFiles] = await Promise.all([
 			axios.get(`https://ydaemon.yearn.finance/${_chainID}/vaults/all?classification=any&strategiesRisk=withRisk`),
 			axios.get('https://raw.githubusercontent.com/LedgerHQ/app-plugin-yearn/develop/tests/yearn/b2c.json'),
-			axios.get('https://raw.githubusercontent.com/yearn/yearn-data-analytics/master/src/risk_framework/risks.json')
-		]) as [any, any, any];
+			axios.get('https://raw.githubusercontent.com/yearn/yearn-data-analytics/master/src/risk_framework/risks.json'),
+			axios.get(`https://api.github.com/repos/yearn/yearn-meta/contents/data/vaults/${_chainID}`)
```

Comment:
Meta has been merged with yDaemon, we should use the new URL here!
```suggestion
			axios.get(`https://api.github.com/repos/yearn/ydaemon/contents/data/meta/vaults/${_chainID}`)
```